app/routes/quotes/QuotesRoute.js

```diff
@@ -9,24 +9,26 @@ import {
 } from 'app/actions/QuoteActions';
 import QuotePage from './components/QuotePage';
 import prepare from 'app/utils/prepare';
-import {
-  selectSortedQuotes,
-  selectCommentsForQuotes
-} from 'app/reducers/quotes';
+import { selectSortedQuotes } from 'app/reducers/quotes';
 import { LoginPage } from 'app/components/LoginForm';
 import replaceUnlessLoggedIn from 'app/utils/replaceUnlessLoggedIn';
 import { selectPagination } from '../../reducers/selectors';
-import { deleteComment } from 'app/actions/CommentActions';
+import { addReaction, deleteReaction } from 'app/actions/ReactionActions';
+import { selectEmojis } from 'app/reducers/emojis';
+import { fetchEmojis } from 'app/actions/EmojiActions';
 
 const mapStateToProps = (state, props) => {
   const queryString = ['?approved=true', '?approved=false'];
   const showFetchMore = selectPagination('quotes', { queryString })(state);
+  const emojis = selectEmojis(state);
   return {
     quotes: selectSortedQuotes(state, props.location.query),
     query: props.location.query,
     actionGrant: state.quotes.actionGrant,
     showFetchMore,
-    comments: selectCommentsForQuotes(state, props)
+    emojis,
+    fetching: state.quotes.fetching,
```

Comment:
Cleaner to have this between `actionGrant: state.quotes.actionGrant` and `showFetchMore`.